app/models/concerns/user/user_media_seen.rb

```diff
@@ -0,0 +1,14 @@
+class User < ActiveRecord::Base
+  module UserMediaSeen
+
+    def media_seen(media_id)
+      userMedia = UserSeenImage.where(user_id: id, content_media_id: media_id)
+      if userMedia.empty?
```

Comment:
puedas usar `unless` y evitar el `if else`
`unless foo?`
`  to so something`
`end`

`unless` es igual a `!if`